src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = (
+  { person },
+) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+    <p className="Person__age">
+      {`I am ${person.age}`}
+    </p>
+    )}
+
+    <p className="Person__partner">
+      {person.isMarried
+        ? (
+          `${person.partnerName} is my `
+            + `${person.sex === 'f' ? 'husband' : 'wife'}`
+        )
+        : (
+          'I am not married'
+        )
+        }
```

Comment:
generate this string out of jsx, create some function or just spit it into a few variables with ternary operators